protocols/imap/src/test/java/org/apache/james/imap/processor/base/MailboxEventAnalyserTest.java

```diff
@@ -176,6 +177,11 @@ public MailboxSession login(String userid, String passwd, Logger log) throws Mai
         public MessageManager getMailbox(MailboxPath mailboxPath, MailboxSession session) throws MailboxException {
             return new MessageManager() {
 
+                @Override
```

Comment:
This changed should belong to this commit: "Offer direct UID retrieval from MessageManager "